cluster-up/cluster/kind-k8s-1.17.0-ipv6/provider.sh

```diff
@@ -11,4 +13,25 @@ source ${KUBEVIRTCI_PATH}/cluster/kind/common.sh
 function up() {
     cp $KIND_MANIFESTS_DIR/kind-ipv6.yaml ${KUBEVIRTCI_CONFIG_PATH}/$KUBEVIRT_PROVIDER/kind.yaml
     kind_up
+    # remove the rancher.io kind default storageClass
+    _kubectl delete sc standard
+
+    nodes=$(_kubectl get nodes -o=custom-columns=:.metadata.name | awk NF)
+    for node in $nodes; do
+        # Create local-volume directories
+        for i in {1..10}; do
+            node=$node mount_disk $i
+        done
+        $DOCKER exec -it $node bash -c "chmod -R 777 /var/local/kubevirt-storage/local-volume"
+    done
+
+    # create the `local` storage class - which functional tests assume to exist
+    _kubectl apply -f $KIND_MANIFESTS_DIR/local-volume.yaml
```

Comment:
Any reason not to use the existing one in cluster-provision/manifests/local-volume.yaml

Reply:
So from what I understand, there is a provision step that builds the images, and in that step that file is copied onto the nodes. And during the cluster startup the yaml is applied from the node. But this is in the VM case, I guess kind is different?

Reply:
Yes, seems we just fetch the `kind` that k8s-sigs provides: https://github.com/kubevirt/kubevirtci/blob/master/cluster-up/cluster/kind/common.sh#L35

Reply:
Interesting. Well I simply want to avoid having the same file in 2 places so we don't over time get drift between the two. I don't care that much where it lives.

Reply:
At kubevirtci we are doing transition on having providers isoletad, for easy debugging and releasing, even if it means duplication, in case of running needed is much better, and we don't have that much.